.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+
+updates:
+
+  # Update npm packages
+  - package-ecosystem: npm
+    directory: /
+    reviewers:
+      - alphagov/design-system-developers
+    schedule:
+      interval: daily
+    allow:
+      - dependency-type: all
+
+    ignore:
+      # Ignore major/minor updates (Marked parser changes output)
+      - dependency-name: jstransformer-marked
```

Comment:
Added `jstransformer-marked` to **ignore** to explain why we only want patch updates (see https://github.com/alphagov/govuk-design-system/pull/2405)